srcpkgs/dendrite/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'dendrite'
+pkgname=dendrite
+version=0.6.3
+revision=1
+build_style=go
+go_import_path="github.com/matrix-org/dendrite"
+go_package="./cmd/create-account ./cmd/dendrite-monolith-server
+ ./cmd/dendrite-polylith-multi ./cmd/generate-config ./cmd/generate-keys"
```

Comment:
This looks like we'd end up with `create-account`, `generate-config` and `generate-keys` as binaries in `/usr/bin` in the end. If that's accurate, we should really, really, really avoid that.

Aside of that: Splitting those out into subpackages would be worth it IMO, as nearly no one will be needing the `dendrite-polylith-multi` binary.

Reply:
I think, since `dendrite-polylith-multi` requires the unpackaged NATS, I'll just not build it for now since I don't use it.

For the utilities (`create-account` et al), I prepended them with `dendrite-` to make them less ambiguous.